playbooks/roles/nghttpx/vars/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+nghttpx_path: "/etc/nghttpx"
+nghttpx_config_file: "{{ nghttpx_path }}/nghttpx.conf"
+
+nghttpx_days_valid: "1825"
+
+# The package name list for Ubuntu 16.04 is from official README.
+# But all we need is nghttpx, so we could drop some of them (commented out).
+nghttpx_dependencies:
+  - g++
+  - make
+  - binutils
+  - autoconf
+  - automake
+  - autotools-dev
+  - libtool
+  - pkg-config
+  - zlib1g-dev
+  - libcunit1-dev
+  - libssl-dev
+  #- libxml2-dev
+  - libev-dev
+  #- libevent-dev
+  #- libjansson-dev
+  - libc-ares-dev
+  - libjemalloc-dev
+  - libsystemd-dev
+  #- libspdylay-dev
+  #- cython
+  #- python3-dev
+  #- python-setuptools
+  - gnutls-bin  # certtool
+
```

Comment:
IMHO the Streisand project itself is a good-written cookbook of Ansible, isn't it? People write comments in code, so in the future it's easier to get the idea what does the code do. These lines are commented out by intention.